src/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 function normalizeInput(fileInput, extension) {
-	if (!Array.isArray(fileInput)) fileInput = [fileInput];
+	if (!Array.isArray(fileInput)) fileInput = [ fileInput ];
```

Comment:
There are bunch of these unrelated formatting changes here. I don't consider these a blocker, but would be happier if they weren't here, as they make PR & commit diffs needlessly messy :upside_down_face: Maybe you have some editor setting or plugin, like Prettier plugin, that's adding these on save?

Reply:
Really having a fight with my autoformatter over this one sorry, will remove the unnecessary changes